packages/flutter/lib/src/widgets/actions.dart

```diff
@@ -1039,29 +1224,58 @@ class _ActionsState extends State<Actions> {
     }
     listenedActions = widgetActions;
   }
+  
+  void _registerActions() => registry?.registerActions(registeredActionSubtreeKey);
+
+  void _unregisterActions() => registry?.unregisterActions(registeredActionSubtreeKey);
+
+  @override
+  void didChangeDependencies() {
+    super.didChangeDependencies();
+    registry = Actions.maybeRegistryOf(context);
```

Comment:
Does this widget need to depend on the registry even if `registerActions` is false?

Reply:
No.  Fixed.